README.md

```diff
@@ -183,37 +183,23 @@ module.exports = {
 #### Strict Mode
 
 `"use strict"` directives in every code block would be annoying.
-The `plugin:markdown/recommended` config enables the [`impliedStrict` parser option](https://eslint.org/docs/user-guide/configuring#specifying-parser-options) and disables the [`strict` rule](https://eslint.org/docs/rules/strict) in Markdown files.
+The `markdown.configs.recommended` config enables the [`impliedStrict` parser option](https://eslint.org/docs/user-guide/configuring#specifying-parser-options) and disables the [`strict` rule](https://eslint.org/docs/rules/strict) in Markdown files.
 This opts into strict mode parsing without repeated `"use strict"` directives.
 
 #### Unsatisfiable Rules
 
 Markdown code blocks are not real files, so ESLint's file-format rules do not apply.
-The `plugin:markdown/recommended` config disables these rules in Markdown files:
+The `markdown.configs.recommended` config disables these rules in Markdown files:
 
 - [`eol-last`](https://eslint.org/docs/rules/eol-last): The Markdown parser trims trailing newlines from code blocks.
 - [`unicode-bom`](https://eslint.org/docs/rules/unicode-bom): Markdown code blocks do not have Unicode Byte Order Marks.
 
 ### Running
 
-#### ESLint v8+
-
 If you are using an `eslint.config.js` file, then you can run ESLint as usual and it will pick up file patterns in your config file. The `--ext` option is not available when using flat config.
 
 If you are using an `.eslintrc.*` file, then you can run ESLint as usual and it will work as in ESLint v7.x.
```

Comment:
```suggestion
If you are using an `.eslintrc.*` file, then you can run ESLint as usual and it will pick up file extensions specified in `overrides[].files` patterns in config files.
```

Since we're removing the `ESLint v7` section below, it seems better to copy the explanation here than to refer to v7 behavior.